locales/en.ini

```diff
@@ -13,6 +13,7 @@ qa-intro = So you enjoy finding other programmers' mistakes?
 webqa-extra = find problems in Mozilla's websites
 ffqa-extra = help us find bugs in Firefox
 thunderbirdqa-extra = help us find bugs in Thunderbird
+oneanddone-extra = learn about Mozilla QA
```

Comment:
This is pretty vague. Is there anything more specific we can put here that will help newcomers understand what it is they'll be linked to?